src/javascripts/transifex-api/resource.js

```diff
@@ -142,32 +146,34 @@ var resource = module.exports = {
         io.popSync(resource.key + resourceName + languageCode);
         this.store(resource.key + resourceName, jqXHR.status);
         io.opSet(resourceName, 'fail');
-        this.checkAsyncComplete();
```

Comment:
No reason to check for sync completion at this point since we upload resources one at a time (so we will end up getting multiple notifications).